<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgetFaceRec">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabFaceRec">
       <attribute name="title">
        <string>FaceRecognition</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBoxSelecTxtP">
          <property name="title">
           <string>选择的Txt路径</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="labelFRSelectTxtPath">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>341</width>
             <height>191</height>
            </rect>
           </property>
           <property name="plainText">
            <string>AddTrainedTxtFile按钮将选择身份人脸图片信息文件txt所在的路径,用于身份识别
Recognize将开始进行人脸识别和身份识别
Stop将关闭摄像头,停止进程

txt文件格式(标签是一样的)
图片1绝对路径
标签
图片2绝对路径
标签
</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QGroupBox" name="groupBoxTrainSetP">
          <property name="title">
           <string>数据集根目录</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QLabel" name="labelFRTrainSetRootPath">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>人脸识别+身份识别(通过级联分类器的XML文件识别身份)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBoxGenTxtP">
          <property name="title">
           <string>生成的Txt路径</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QLabel" name="labelFRGenTxtPath">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QGroupBox" name="groupBoxGenXMLP">
          <property name="title">
           <string>生成的XML路径</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="labelFRGenXMLPath">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QGroupBox" name="groupBoxSelecXMLP">
          <property name="title">
           <string>选择的XML路径</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QLabel" name="labelFRSelectXMLPath">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QGroupBox" name="groupBoxTOOLS">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonFRRunRec">
             <property name="text">
              <string>Recognize</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButtonFRStop">
             <property name="text">
              <string>STOP</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButtonFRGenTxt">
             <property name="text">
              <string>生成txt文件</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pushButtonFRSelectXML">
             <property name="text">
              <string>选择XML文件</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButtonFRSelectTxt">
             <property name="text">
              <string>选择txt文件</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="pushButtonFRGenXML">
             <property name="text">
              <string>生成XML文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QGroupBox" name="groupBox_13">
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string/>
          </property>
          <widget class="QPlainTextEdit" name="plainTextEditTrainSetFileNames">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>351</width>
             <height>192</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="0" rowspan="7" colspan="2">
         <widget class="QLabel" name="labelShowFace">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabObjDetect">
       <attribute name="title">
        <string>ObjectDetect</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>目标检测,暂时只做目标匹配方面</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBoxSrc">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QLabel" name="labelShowODSrc">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>271</width>
             <height>216</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QGroupBox" name="groupBoxTemplate">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <widget class="QLabel" name="labelShowODTemp">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>271</width>
             <height>216</height>
            </rect>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2" colspan="2">
         <widget class="QLabel" name="labelShowODResult">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" rowspan="2">
         <widget class="QGroupBox" name="groupBoxButtons">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButtonODMatch">
             <property name="text">
              <string>Match</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButtonODAddSrc">
             <property name="text">
              <string>AddSourceImage</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButtonODAddTemp">
             <property name="text">
              <string>AddTemplateImage</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabBankCardRec">
       <attribute name="title">
        <string>BankCardRec</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="2">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>银行卡识别,基于形态学(卡面需固定)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="labelShowBCSrc">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>GroupBox</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2" colspan="3">
         <widget class="QLabel" name="labelsShowBCRecResult">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" rowspan="2">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>CNN</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1145</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
